README.md

```diff
@@ -493,6 +493,12 @@ name,iterations,real_time,cpu_time,bytes_per_second,items_per_second,label
 "BM_SetInsert/1024/10",106365,17238.4,8421.53,4.74973e+06,1.18743e+06,
 ```
 
+## Output Files
+The library supports writing the output of the benchmark to a file specified
+by `--benchmark_output=<filename>`. The format of the output can be specified
+using `--benchmark_output_format={json|console|csv}`. Specifying
+`--benchmark_output` does not suppress the console output.
```

Comment:
ok, i'll buy that. bikeshedding then:

--benchmark_out: the file to which to write results
--benchmark_out_format: the format in which to write the results to --benchmark_out, if set
--benchmark_format: the format in which to write the results to stdout

?